include/benchmark/benchmark.h

```diff
@@ -1,386 +1,45 @@
-// Support for registering benchmarks for functions.
-
-/* Example usage:
-// Define a function that executes the code to be measured a
-// specified number of times:
-static void BM_StringCreation(benchmark::State& state) {
-  while (state.KeepRunning())
-    std::string empty_string;
-}
+// Copyright 2014 Google Inc. All rights reserved.
```

Comment:
2015 :) (if we need a copyright notice at all, which i believe we no longer do.)